proposals/collection-literals.md

```diff
@@ -112,23 +112,46 @@ Collection literals are [target-typed](https://github.com/dotnet/csharplang/blob
 
     * The behavior of collection literals with collections that are not well-behaved is undefined.
 
-## Constructible collection types (target types)
-[constructible-collection-types]: #constructible-collection-types
+## Conversions
+A *collection literal conversion* allows a collection literal expression to be converted to a type.
 
-The following are _constructible collection types_ that can be used to construct collection literals.
-These are the valid _target types_ for collection literals.
+The following implicit *collection literal conversions* exist from a collection literal expression:
 
-The list is in priority order. If a collection type belongs in multiple categories, the first is used.
+* To a single dimensional *array type* `T[]`, or a *span type* `System.Span<T>` or `System.ReadOnlySpan<T>`, where for each element `Ei`:
```

Comment:
Specese clarification: you define `Ei` for this section, then use it for all the rest of the sections. You probably want to define it above.